documentation/docs/agent-configuration/setup-library-agent.md

```diff
@@ -0,0 +1,158 @@
+import { getResolver as ethrDidResolver } from 'ethr-did-resolver';
+import { getResolver as webDidResolver } from 'web-did-resolver';
+
+//veramo data store
+import { Entities, KeyStore, DIDStore, PrivateKeyStore, migrations } from '@veramo/data-store'
+
+//typeorm
+import { DataSource } from 'typeorm'
+```
+## Variables
+Create some variables in your set up file, remember to replace their value by yours.
+```typescript
+const INFURA_PROJECT_ID = '<YOUR-INFURA-PROJECT-ID>';
+const DB_SECRET_KEY ='<YOUR-GENERATED-SECRET-KEY>';
+
+const DATABASE_FILE = 'database.sqlite'
+```
+## Initialize the agent
+This is how an agent is initialized in your set up file, copy and paste into your file.
+```typescript
+type InstalledPlugins = IResolver &
+    IKeyManager &
+    IDIDManager &
+    ICredentialPlugin &
+    IDataStoreORM &
+    IDataStore &
+    IMessageHandler&
+    IRenderer;
+
+const dbConnection = new DataSource({
+    type: 'sqlite',
+    database: DATABASE_FILE,
+    synchronize: false,
+    migrations,
+    migrationsRun: true,
+    logging: ['error', 'info', 'warn'],
+    entities: Entities,
+}).initialize()
+
+export const agent: TAgent<InstalledPlugins> = createAgent<InstalledPlugins>({
+    plugins: [
+        new DIDResolverPlugin({
+            resolver: new Resolver({
+                ...ethrDidResolver({ infuraProjectId: INFURA_PROJECT_ID }),
+                ...webDidResolver(),
+                ...getDidKeyResolver(),
+                ...getDidPkhResolver(),
+                ...getDidJwkResolver(),
+            }),
+        }),
+        new KeyManager({
+            store: new KeyStore(dbConnection),
+            kms: {
+                local: new KeyManagementSystem(new PrivateKeyStore(dbConnection, new SecretBox(DB_SECRET_KEY))),
+                web3: new Web3KeyManagementSystem({}),
+            },
+        }),
+        new DIDManager({
+            store: new DIDStore(dbConnection),
+            defaultProvider: 'did:ethr:goerli',
+            providers: {
+                'did:ethr': new EthrDIDProvider({
+                    defaultKms: 'local',
+                    ttl: 60 * 60 * 24 * 30 * 12 + 1,
+                    networks: [
+                        {
+                            name: 'mainnet',
+                            rpcUrl: 'https://mainnet.infura.io/v3/' + INFURA_PROJECT_ID,
+                        },
+                        {
+                            name: 'goerli',
+                            rpcUrl: 'https://goerli.infura.io/v3/' + INFURA_PROJECT_ID,
+                        },
+                        {
+                            chainId: 421613,
+                            name: 'arbitrum:goerli',
+                            rpcUrl:
+                                'https://arbitrum-goerli.infura.io/v3/' + INFURA_PROJECT_ID,
+                            registry: '0x8FFfcD6a85D29E9C33517aaf60b16FE4548f517E',
+                        },
+                    ],
+                }),
```

Comment:
Currently, the vckit supports only `did:web` and `did:key`, so we should remove the rest